plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/DataBoundConfigurator.java

```diff
@@ -24,8 +24,10 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+import java.util.logging.Level;
 import java.util.logging.Logger;
 import javax.annotation.Nonnull;
+import javax.annotation.ParametersAreNonnullByDefault;
```

Comment:
Does this one not fall under JSR 305?
https://blog.codefx.org/java/jsr-305-java-9/#comment-3582702088
Would it not be better to use one from SpotBugs?
https://github.com/spotbugs/spotbugs/blob/master/spotbugs-annotations/src/main/java/edu/umd/cs/findbugs/annotations/ReturnValuesAreNonnullByDefault.java

Reply:
I'm not sure what you mean here. `ReturnValuesAreNonnullByDefault` seems to mean that the functions will never return `null`.

Reply:
Just concerned about relying on JSR 305 annotations.

Reply:
I think it could be a separate improvement issue, applies to `@Nonnull` also. Maybe not only SpotBugs, but also other common annotation types.